### PR TITLE
Feature/product image error

### LINKED ISSUE #104: image icon used when there is no bad image
Explanation

When entering the product detail and the user does not have a photo of that product, the image that is in the design must appear and not the white logo

**Current**
imagen

**Expected result**
imagen

### PR DESCRIPTION
**Pull request by eduardoquispe**

### CODE REVIEW

File: src/components/DetailLayout/DetailLayout.vue [hunk 1]
```diff
@@ -24,4 +29,7 @@ interface DetailLayoutProps {
 }
 
 defineProps<DetailLayoutProps>();
+
+const { name } = useAttrs();
+console.log(name);
```
Comment (Eghinner) [inline src/components/DetailLayout/DetailLayout.vue]: 🦖

File: src/components/DetailLayout/styles.scss [hunk 2]
```diff
@@ -27,6 +27,22 @@
       padding: 1.25rem;
       object-fit: contain !important;
     }
+
+    &--error {
+      background-color: #000 !important;
```
Comment (Eghinner) [inline src/components/DetailLayout/styles.scss]: its necessary `!important`?

File: src/pages/Product/Product.layout.vue [hunk 3]
```diff
@@ -29,7 +29,7 @@
 
 <script lang="ts" setup>
 import DetailLayout from 'src/components/DetailLayout';
-import DIDLogo from 'src/assets/logos/didperu.svg';
```
Comment (Eghinner) [inline src/pages/Product/Product.layout.vue:32]: could you check if the old image is still in use? if not, delete it.

Comment (eduardoquispe) [reply src/pages/Product/Product.layout.vue:32]: @Eghinner Yes it is still used

**Comment by eduardoquispe** [comment]
@Eghinner updated

**Review by Eghinner — APPROVED** [review]

**Review by Eghinner — APPROVED** [review]

File: src/pages/Product/Product.layout.vue (hunk 3 above)
Comment (Eghinner) [reply src/pages/Product/Product.layout.vue:32]: ok

**Comment by angelik0828** [comment]
## ✔️ Change made, when entering the product detail view when there is no image, the correct image showing the design appears.
image

**Review by angelik0828 — APPROVED** [review]

**Pull request merged**